docs/static/releasenotes.asciidoc

```diff
@@ -68,6 +69,82 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-16-0]]
+=== Logstash 8.16.0 Release Notes
+
+==== Announcing the new {ls} Health Report API
+
+The new Health Report API (`GET /_health_report`) is available starting with {ls} `8.16.0`.
+This API leverages indicators capable of detecting the degraded status of pipelines and
+providing actionable insights https://github.com/elastic/logstash/pull/16520[#16520], https://github.com/elastic/logstash/pull/16532[#16532].
+
+NOTE for users upgrading to {ls} 8.16.0: if your existing automation relies on liveliness scripts that expect Logstash's API status to either not be available or return a hardcoded `green` status, there is an escape hatch to prevent the new Health API's status from affecting the top-level `status` field of Logstash's existing API responses. Setting the `logstash.forceApiStatus` system property to `green` will force other APIs to return the previous hard-coded `green` value. https://github.com/elastic/logstash/pull/16535[#16535]
```

Comment:
```suggestion
If your {ls} instances use automation that relies on pre-8.16.0 behavior of API statuses, you can maintain the current behavior and prevent the new Health API status from affecting the top-level `status` field of existing {ls} API responses. 
You can add the `logstash.forceApiStatus: green` system property to the `logstash-yml` config file.
This setting forces other APIs to return the previous hard-coded `green` value. https://github.com/elastic/logstash/pull/16535[#16535]
```

Reply:
Is it necessary to call out specific situations where this change might affect automations, or can we generalize as suggested?  I'm wondering how many users this will affect, and also thinking that uses affected would know based on the reduced description. Let's discuss. 

Also, we need to be specific about the location where the property should be added. I guessed. 
Let's discuss.

Reply:
> Is it necessary to call out specific situations where this change might affect automations, or can we generalize as suggested?

IMO, I'd sticky to the more specific version. This change can cause troubles for users relying on that green status, specially the one using it for liveliness purposes. 

> Also, we need to be specific about the location where the property should be added.

Agreed, this seems to be a system property so I guess the correct place would be the `jvm.options` file, using the following format: `-Dlogstash.forceApiStatus=green`

@yaauie, @jsvd?

Reply:
> Agreed, this seems to be a system property so I guess the correct place would be the jvm.options file, using the following format: -Dlogstash.forceApiStatus=green

Correct, from https://github.com/elastic/logstash/pull/16535:

> ### How to test this PR locally
>
> Add the following to your `config/jvm.options`:
> `-Dlogstash.forceApiStatus=green`

Reply:
> IMO, I'd sticky to the more specific version. This change can cause troubles for users relying on that green status, specially the one using it for liveliness purposes.

+++ I'd like to keep a specific example so users understand the real life impact, as it may not be obvious at a first glance.

Reply:
I updated the suggestion, taking discussion points into consideration. 
Please LMKWYT.